samples/command/provider/src/provider_impl.rs

```diff
@@ -121,6 +126,19 @@ impl DigitalTwinProvider for ProviderImpl {
 
         Ok(Response::new(response))
     }
+
+    /// Stream implementation.
+    ///
+    /// # Arguments
+    /// * `request` - OpenStream request.
+    async fn stream(
```

Comment:
I need to separate the gRPC interface for all operations into several different groups: get/set/invoke, streaming, pub/sub, so that we don't see this in each sample.  This refactoring will be done at a later time.